Mu2eG4/src/constructHall.cc

```diff
@@ -240,6 +285,7 @@ namespace mu2e {
     AntiLeakRegistry& reg = art::ServiceHandle<Mu2eG4Helper>()->antiLeakRegistry();
 
     OrientationResolver* OR = new OrientationResolver();
+    OrientationResolver* ORH = new OrientationResolver();
```

Comment:
Same comment as above for new/delete vs making an object on the stack.

Reply:
modified ORH in both functions. I haven't modified the way OR was called.

Reply:
I believe that you only need to have one of these objects, not both.  The class has no state and should have been implemented as a free function.    

Please make the same change to OR that you did to ORH.  Then remove ORH and use OR everywhere.  Can you do that today?

Reply:
Oh! I ignored that, they both are the same. Yes, I will do it today.

Reply:
Thanks.